src/dialect/mod.rs

```diff
@@ -636,6 +636,10 @@ pub trait Dialect: Debug + Any {
         false
     }
 
+    fn supports_dollar_quoted_string(&self) -> bool {
```

Comment:
It's better to add some documents for this method. (Just like others).

Reply:
Added!